src/Makefile.am

```diff
@@ -132,10 +132,14 @@ src_libt8_la_SOURCES = \
 src_libt8_la_CPPFLAGS = $(AM_CPPFLAGS) $(T8_CPPFLAGS)
 ## This is the official API versioning scheme of libtool.  Please see:
 ## Read https://www.gnu.org/software/libtool/manual/libtool.html#Versioning
-src_libt8_la_LDFLAGS = -release $(VERSION) -version-info 2:0:0
+src_libt8_la_LDFLAGS = -version-info 2:0:0
 src_libt8_la_LIBADD = @T8_P4EST_LDADD@ @T8_SC_LDADD@
```

Comment:
This works by accident; with the latest commits on feature-make-install you can use T8_*_LIBADD.

Reply:
Can you elaborate respectively propose a patch? I'm not savvy in autotools and coded by trial-and-error only.